_studio/mfx_lib/fei/h264_common/fei_common.cpp

```diff
@@ -645,7 +645,9 @@ mfxStatus MfxH264FEIcommon::CheckInitExtBuffers(const MfxVideoParam & owned_vide
 {
     // Slice number control through CO3 is not allowed
     mfxExtCodingOption3 const * extOpt3 = GetExtBuffer(owned_video);
+    MFX_CHECK(extOpt3,  MFX_ERR_UNDEFINED_BEHAVIOR);
```

Comment:
owned video is of type MfxVideoParam is ALWAYS creates all ext buffers which could be used for init, so these pointers can't be NULL. I think - best way to get KW happy and not to introduce additional checks - use GetExtBufferRef as you did below

Reply:
Done.